data/build.gradle.kts

```diff
@@ -57,6 +57,7 @@ dependencies {
     api(libs.kotlin.coroutines.datetime)
     implementation(libs.android.hilt)
     implementation(libs.timber)
+    implementation("androidx.paging:paging-common-ktx:3.1.1")
```

Comment:
Kindly check the read me on how to add dependencies and correct this

Reply:
I have updated the dependency according to the read me guidelines

Reply:
Looks good